apps/remix-ide-e2e/src/tests/solidityUnittests.test.ts

```diff
@@ -112,8 +112,8 @@ module.exports = {
       // .click('#singleTesttests/compilationError_test.sol')
       .clickElementAtPosition('.singleTestLabel', 2)
       .scrollAndClick('*[data-id="testTabRunTestsTabRunAction"]')
-      .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', 'SyntaxError: No visibility specified', 120000)
-      .waitForElementContainsText('*[data-id="testTabTestsExecutionStoppedError"]', 'The test execution has been stopped because of error(s) in your test file', 120000)
+      .waitForElementContainsText('*[data-id="testTabSolidityUnitTestsOutput"]', 'SyntaxError: No visibility specified', 60000)
+      .waitForElementContainsText('*[data-id="testTabTestsExecutionStoppedError"]', 'The test execution has been stopped because of error(s, 60000) in your test file', 60000)
```

Comment:
timeout update not required  in `The test execution has been stopped because of error(s, 60000) in your test file`